README.md

```diff
@@ -1,4 +1,4 @@
-# ros-system-monitor
+# system_monitor
```

Comment:
Why did you change the package name? I know that is not how we named the packages on our repos, but this is a fork, shouldn't we keep the original naming?

Reply:
The original name of the package in `cmake` and `package.xml` is __system-monitor__ and the project name in the repository is __ros-system-monitor__. In my view it makes sense but both have the same name.

![image](https://user-images.githubusercontent.com/10222007/80736578-b6ac1c80-8ae8-11ea-82b8-498f8bdc9398.png)

Reply:
It is true, I agree